src/ILCompiler.ReadyToRun/src/Compiler/ReadyToRunCodegenCompilation.cs

```diff
@@ -25,6 +25,8 @@ public abstract class Compilation : ICompilation
         protected readonly Logger _logger;
         private readonly DevirtualizationManager _devirtualizationManager;
         private ILCache _methodILCache;
+        private readonly List<ModuleDesc> _modulesBeingInstrumented;
```

Comment:
How many modules do typically get instrumented? If it can be the whole program, we probably want a HashSet.

Reply:
Eh, possibly. The current count is always 1, but we may have higher counts in the future. I'll just make it a HashSet as it won't hurt, and might be beneficial.

---
In reply to: [321142266](https://github.com/dotnet/corert/pull/7758#discussion_r321142266) [](ancestors = 321142266)